### PR TITLE
Update plugin-shell.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Minor tweaks.

### CODE REVIEW

File: en/console-and-shells/plugin-shell.rst [hunk 1]
```diff
@@ -18,19 +18,19 @@ Via the `Load` task you are able to load plugins in your
 
 This will add the following to your **config/bootstrap.php**::
 
-    Plugin::load('MyPlugin', []);
+    Plugin::load('MyPlugin');
 
-By adding `-r` or `-b` to your command you can enable to `bootstrap` and
-`routes` value::
+Adding the `-b` or `-r` switch to the load task will enable loading of the plugin's
+`bootstrap` and `routes` values::
```
Comment (bcrowe) [inline en/console-and-shells/plugin-shell.rst]: If we do want to "keep" the highlighting, these will need to be double backticks.

**Comment by jeffblack360** [comment]
markstory: What caused the check failure?

**Comment by markstory** [comment]
I am not sure. The build check was added yesterday so it might have a few wrinkles still.

**Comment by jeffblack360** [comment]
Thank you sir!

**Pull request merged**